MoreLinq.Test/TestingSequence.cs

```diff
@@ -89,9 +94,14 @@ public DisposeTestingSequenceEnumerator(IEnumerator<T> sequence)
 
             public T Current => _sequence.Current;
             object IEnumerator.Current => Current;
-            public bool MoveNext() => _sequence.MoveNext();
             public void Reset() => _sequence.Reset();
 
+            public bool MoveNext()
+            {
+                OnMoveNext?.Invoke(this, EventArgs.Empty);
```

Comment:
The event should be raised _if_ `_sequence.MoveNext()` returned `true`. Otherwise `MoveNextCounter` is misleading is really a counter of unconditional invocations of `MoveNext` on `_sequence`. I don't think that's what's intended.

Reply:
My intention was indeed to count *all* invocations of `MoveNext()`, and not *only* the number of positives when invoking `MoveNext().`
I wanted to verify that `MoveNext()` was invoked *exactly* the minimum number of times necessary.

Reply:
I see. Perhaps then calling it `MoveNextCallCount` would be more accurate?

Reply:
I've  renamed the property and backing field on `TestingSequence` from `MoveNextCount` to `MoveNextCallCount`.